.github/workflows/codeql-analysis.yaml

```diff
@@ -46,11 +46,15 @@ jobs:
       uses: actions/setup-python@98f2ad02fd48d057ee3b4d4f66525b231c3e52b6
       with:
         python-version: ${{ matrix.python }}
```

Comment:
I think we can turn off CodeQL on `py 3.9`. CodeQL takes a lot of time and I'm not sure about the benefit of checking against different Python versions.

Reply:
Commit https://github.com/jenstroeger/python-package-template/pull/248/commits/846ef236ea3fc0b54b93fd657972ff6e4b051c0d.